skeletons/INTEGER.h

```diff
@@ -43,6 +43,8 @@ xer_type_encoder_f INTEGER_encode_xer;
 per_type_decoder_f INTEGER_decode_uper;
 per_type_encoder_f INTEGER_encode_uper;
 
+#define INTEGER_constraint asn_generic_no_constraint
```

Comment:
> Do you mean you use (1) your own repository to generate codes for sample.source.PKIX1 ?

Yes that's exactly what I meant. And I posted the result of running the freshly-generated `x509dump` on the provided/included `sample-Certificate-1.der`.

> Please remember to execute `make regen` to re-generate these C files and H files, otherwise, the existed files will be the same

For me `make distclean` does the job. :-)
 (Unless I'm missing something?).